packages/server/test/unit/config_spec.coffee

```diff
@@ -347,6 +347,15 @@ describe "lib/config", ->
           @setup({video: 42})
           @expectValidationFails("be a boolean")
 
+      context "videoUpload", ->
+        it "passes if a boolean", ->
+          @setup({videoUpload: false})
+          @expectValidationPasses()
+
+        it "fails if not a boolean", ->
+          @setup({videoUpload: 99})
+          @expectValidationFails("be a boolean")
+
```

Comment:
should we add a test for undefined? Or that's impossible in typescript? Something like

```suggestion
        it "fails if undefined", ->
          @setup({videoUpload: undefined})
          @expectValidationFails("be a boolean")

```